packages/examples/packages/name-lookup/src/index.test.ts

```diff
@@ -45,7 +49,9 @@ describe('onNameLookup', () => {
       chainId: CHAIN_ID_MOCK,
     } as any;
 
-    expect(await onNameLookup(request)).toStrictEqual({
+    const { onNameLookup } = await installSnap();
```

Comment:
We can remove this test case now since it is not possible to test due to validation. 

We validate the request here: https://github.com/MetaMask/snaps/blob/main/packages/snaps-execution-environments/src/common/commands.ts#L60

Reply:
Thanks for clarification! Error message was difficult to understand (where the validation is coming from and why it was failing).

Removed.